versions/f-/fontconfig.json

```
@@ -1,7 +1,12 @@
 {
   "versions": [
     {
-      "git-tree": "de83a21d912d9b87d8ad069d94dffcca201830a3",
+      "git-tree": "e4934e51612999eea86c66bcc794fa88aedef173",
+      "version-string": "2.13.1",
+      "port-version": 6
+    },
+    {
+      "git-tree": "d9774e3ba72153356e6ba148678304283053feda",
```

```suggestion
      "git-tree": "de83a21d912d9b87d8ad069d94dffcca201830a3",
```
Previous versions must not be modified.